src/test-run/commands/index.js

```diff
@@ -63,14 +68,23 @@ function positiveIntegerArgument (name, val) {
         throw new ActionPositiveIntegerArgumentError(name, val);
 }
 
-function nonEmptyStringArgument (option, val) {
+function nonEmptyStringArgument (name, val) {
     var type = typeof val;
 
     if (type !== 'string')
-        throw new ActionStringArgumentError(option, type);
+        throw new ActionStringArgumentError(name, type);
 
     if (!val.length)
-        throw new ActionStringArgumentError(option);
+        throw new ActionStringArgumentError(name);
+}
+
+function navigateToArgument (name, val) {
+    nonEmptyStringArgument(name, val);
+
+    var url = val.trim();
+
+    if (PROTOCOL_RE.test(url) && !SUPPORTED_PROTOCOL_RE.test(url))
```

Comment:
I think single regex will be too complicated.
I suggest rewrite it:
```
function navigateToUrlArgument (name, val) {
    nonEmptyStringArgument(name, val);

    var url      = val.trim();
    var protocol = PROTOCOL_RE.match(url);

    if (protocol && !SUPPORTED_PROTOCOL_RE.test(protocol))
        throw new ActionUnsupportedProtocolError(name, protocol);
}
```
And use `protocol` in errror's text:
`The <code>${err.argumentName}</code> argument specifies a URL that uses an unsupported <code>err.protocol</code> protocol. Only HTTP and HTTPS are supported, as well as protocol-relative and relative URLs.`
What do you think @inikulin ?